### PR TITLE
Feature/add site content

### PR DESCRIPTION
**Pull request by amandujano**
new in this branch: 

* adding initial translate pages
* minor style changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
-  Free Robotics web page
+  air4childrem web page
 </h1>
 
 The main intention of this repository is to provide relevant information of the project `Free Robotics`.
 
 ## Getting started
-This is a Gatsby.js web development. In order to run this code you have to install the following: 
-
+This is a Gatsby.js web development. In order to run this code you have type the following commands in console:
```
Comment (mxochicale) [inline README.md:9]: Probably prerequisites section can be added such as the OS distro version in the machine

File: README.md [hunk 2]
```diff
@@ -20,9 +19,18 @@ This is a Gatsby.js web development. In order to run this code you have to insta
    `npm install -g gatsby-cli`
 1. Download this repository   
 1. Have a text editor installed, i.e. [VSCode](https://code.visualstudio.com/)
```
Comment (mxochicale) [inline README.md:21]: Maybe you can add 
Clone this repository
git clone https://github.com/air4children/site
Also, the recommended text editor VSCode but alternatively one can use favorite text editor (e.g , atom, pycharm, etc).

File: README.md [hunk 3]
```diff
@@ -20,9 +19,18 @@ This is a Gatsby.js web development. In order to run this code you have to insta
...
-1. `npm i gatsby-plugin-webpack-bundle-analyzer`
\ No newline at end of file
+1. `npm i gatsby-plugin-webpack-bundle-analyzer`
+1. `npm install gh-pages --save-dev`
+
+### Website color code
+
+Here's an image of the scheme color used in this website:
+
+![color scheme](.docs/color_code.png)
```
Comment (mxochicale) [inline README.md:34]: This looks great!

File: README.md [hunk 4]
```diff
@@ -1,12 +1,11 @@
 <h1 align="center">
-  Free Robotics web page
+  air4childrem web page
```
Comment (mxochicale) [inline README.md:2]: just a spelling error with children

File: src/components/header.en.js [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+  &:last-of-type {
+    margin-right: 0;
+  }
+`;
+
+const Header = () => {
+  const { image } = useStaticQuery(graphql`
+    query {
+      image: file(relativePath: { eq: "air4childrenLogo1.png" }) {
```
Comment (mxochicale) [inline src/components/header.en.js:32]: Maybe you can update the logo with the latest version available here https://github.com/air4children/logos

File: src/pages/videos.mdx [hunk 6]
```diff
@@ -0,0 +1,23 @@
+# Videos
```
Comment (mxochicale) [inline src/pages/videos.mdx:1]: Nice section, perhaps we do not need new lines.

File: README.md (hunk 2 above)
Comment (amandujano) [reply README.md]: Yeah, thanks for this siggestion. I added [here](https://github.com/air4children/site/pull/5/commits/c988c4901982b6a22b7797df5424fcafa03b9a70) more information.

File: README.md (hunk 1 above)
Comment (amandujano) [reply README.md]: Check this [commit](https://github.com/air4children/site/pull/5/commits/4dca893d99a4acf8f99525366828e8c30a65ce32). I added a more descriptive information.

File: README.md (hunk 4 above)
Comment (amandujano) [reply README.md]: Also fixed [here](https://github.com/air4children/site/pull/5/commits/4dca893d99a4acf8f99525366828e8c30a65ce32)

File: src/components/header.en.js (hunk 5 above)
Comment (amandujano) [reply src/components/header.en.js:32]: oh nice. I updated the logo [here](https://github.com/air4children/site/pull/5/commits/8736298340f859e3d5b4310c3e047a795a6dc78e), but I am not using any images by now.

**Comment by amandujano** [comment]
> Hi @amandujano
> Such great job you have made. I just added few comments that are mainly cosmetic ones and the update of some logo of air4children. Perhaps, it is important to put a README for permission of other images that we do not own. By the way is "package-lock.json" a temporal file? Maybe we do not needed it as well and we can add a gitignore for that one and others.

Hey @mxochicale . I added some fixes with  your suggestions. You're right about the images, I am not using images by now, so I didnt use any reference for the permission. 

I was also thinking about the type of Licence the project would have. Do you have some guidance in this topic?

File: src/components/header.en.js (hunk 5 above)
Comment (mxochicale) [reply src/components/header.en.js:32]: Thanks Angel, for pointed out that. It is good news as logo will still change : )

**Comment by amandujano** [comment]
yep. I thinks is ready to be merged :)

**Pull request merged**